include/tpm.h

```diff
@@ -176,6 +175,7 @@ typedef struct efi_tpm2_protocol efi_tpm2_protocol_t;
 
 typedef UINT32                     TCG_EVENTTYPE;
 
+#define EV_IPL                              0x0000000d
```

Comment:
**TCG PC Client Platform Firmware Profile, Family “2.0”, Level 00 Revision 1.04, June 3, 2019** says...

"This event is deprecated."

Reply:
I should say, I like this change to define a magic number... just observing that we're adding a measurement using a deprecated type (should we stick with this for consistency, or choose a different type)?

Reply:
It's actually just /moving/ that definition... and actually using it where we're already using that value.  

That said, I think the point @mjg59 made elsewhere about not changing the event log formats we're already using remains valid here, even though this is a new item being logged.  I think we should stick with this for now: I don't want to come up with a plan on how to transition to different event log entry types ad-hoc in the middle of doing something else.